test/models/story_test.rb

```diff
@@ -26,6 +26,29 @@
     end
   end
 
+  describe 'distributions' do
+    let(:series) { create(:series) }
+    let(:story) { create(:story, series: series) }
+
+    it 'creates story distributions' do
+      stub_request(:post, 'https://id.prx.org/token').
+        to_return(status: 200,
+                  body: '{"access_token":"abc123","token_type":"bearer"}',
+                  headers: { 'Content-Type' => 'application/json; charset=utf-8' })
+
+      stub_request(:get, 'https://feeder.prx.org/api/v1/podcasts/23').
+        to_return(status: 200, body: json_file(:podcast), headers: {})
+
+      stub_request(:post, 'https://feeder.prx.org/api/v1/podcasts/23/episodes').
```

Comment:
you could say that - it throws an error so the test errors out if you don't stub the request.